hardware/raspberrypi/bootmodes/msd.md

```diff
@@ -57,7 +57,8 @@ After five to ten seconds, the Raspberry Pi should begin booting and show the ra
 <a name="pi4"></a>
 ## Raspberry Pi 4
 
-The Raspberry Pi 4's bootcode is stored in [EEPROM](../booteeprom.md) and can be updated. Support for mass storage boot will be added in a future update.
+USB mass storage support is currently undergoing beta testing, once beta test is compelte the release version will enable USB mass storage boot by default and there is no need to set `program_usb_boot_mode`.
```

Comment:
I thought `program_usb_boot_mode` was only for Pi 3B?

Reply:
Precisely, I'm explicitly stating that this isn't required on Pi 4

Reply:
I see. The wording caught me out. I think that bit would be better as a separate sentence, e.g. "The `program_usb_boot_mode` option is not required on the Pi 4."

Reply:
I think it's clear enough already